airflow/ui/src/queries/useTrigger.ts

```diff
@@ -37,14 +36,14 @@ export const useTrigger = ({ dagId, onSuccessConfirm }: { dagId: string; onSucce
   const [dateValidationError, setDateValidationError] = useState<unknown>(undefined);
 
   const onSuccess = async () => {
-    const queryKeys: Array<PartialQueryKey> = [
-      { baseKey: useDagServiceGetDagsKey },
-      { baseKey: useDagsServiceRecentDagRunsKey },
-      { baseKey: useDagRunServiceGetDagRunsKey, options: { dagIds: [dagId] } },
-      { baseKey: useTaskInstanceServiceGetTaskInstancesKey, options: { dagId, dagRunId: "~" } },
+    const queryKeys = [
+      [useDagServiceGetDagsKey],
+      [useDagsServiceRecentDagRunsKey],
+      [useDagRunServiceGetDagRunsKey, { dagId }],
+      [useTaskInstanceServiceGetTaskInstancesKey, { dagId, dagRunId: "~" }],
```

Comment:
Nit: We should use the functions defined to reconstruct the key I think.

This will not break today but could when we upgrade the generation package code later.

Reply:
Fixed!